symplyphysics/laws/astronomy/ratio_of_luminosities_from_ratio_of_masses_of_stars.py

```diff
@@ -23,6 +22,10 @@
 ## L2 - illuminance of second object,
 ## L1 - illuminance of first object.
 
+# Links: last formula in paragraph <https://en.wikipedia.org/wiki/Luminosity#Luminosity_formulae>
+
+# TODO: must be 3.5 and not 4?
```

Comment:
https://en.wikipedia.org/wiki/Mass%E2%80%93luminosity_relation
It depends.
Add proper conditions.

Reply:
Oh, I see now, I'll add a note